app/root.tsx

```diff
@@ -67,6 +71,12 @@ export default function App() {
     }
   }, [location, shouldTrack]);
 
+  useEffect(() => {
+    if (gaTrackingId?.length) {
+      gtag.pageview(location.pathname && gaTrackingId);
```

Comment:
In this line, we are sending the `gaTrackingId` as a prop to `gtag.pageview()` . 
This Id is being sent then to Google through `  ReactGA.send({ hitType: "pageview", page: gaTrackingId });`

Is this what we want? I thought on "page" we are expected to send the subdirectories ("de/ich-suche-redezeit" for example)